src/gz/gz_command.c

```diff
@@ -456,6 +457,13 @@ void command_resetlag(void)
   gz.lag_vi_offset = -(int32_t)z64_vi_counter;
 }
 
+void command_togglewatches(void)
+{
+  settings->bits.watches_visible = !settings->bits.watches_visible;
+  menu_think(gz.menu_main);
+  menu_think(gz.menu_watchlist);
```

Comment:
This line causes an "invalid opcode" error when I press the keybind (R + D-right atm) when I have a watch that's visible on screen outside of the gz menu. Unsure why.

@glankk any ideas?

Reply:
You changed gz.menu_watchlist to a `struct menu_item *` when `menu_think` expects a `struct menu *`

Reply:
I feel silly. Thanks Krim! 😄

Reply:
No problem :)